Any way to cross compile to windows?

OS: Linux debian
Compiler: mingw64
(Lib was downloaded from release)

i try to
```
./configure --host=x86_64-w64-mingw32 --prefix="$(pwd)/_install" --enable-cares --disable-shared --enable-static
```
but got
```
src/conn.c:24:10: fatal error: netinet/in.h: No such file or directory
   24 | #include <netinet/in.h>
      |          ^~~~~~~~~~~~~~
compilation terminated.
```

sorry if this is a dumb question

wow! almost worked!

but now it wants ares.h

```bash
$ make
/bin/bash ./libtool  --tag=CC   --mode=compile x86_64-w64-mingw32-gcc -DPACKAGE_NAME=\"libstrophe\" -DPACKAGE_TARNAME=\"libstrophe\" -DPACKAGE_VERSION=\"0.14.0\" -DPACKAGE_STRING=\"libstrophe\ 0.14.0\" -DPACKAGE_BUGREPORT=\"jack@metajack.im\" -DPACKAGE_URL=\"\" -DPACKAGE=\"libstrophe\" -DVERSION=\"0.14.0\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DSTDC_HEADERS=1 -DLT_OBJDIR=\".libs/\" -DHAVE_SNPRINTF=1 -DHAVE_VSNPRINTF=1 -DHAVE_VA_COPY=1 -DDONT_USE_GETRANDOM=1 -DHAVE_CARES=1 -DHAVE_ZLIB=1 -DLIBXMPP_VERSION_MAJOR=0 -DLIBXMPP_VERSION_MINOR=14 -I.     -I. -Wall -Wextra     -g -O2 -MT src/libstrophe_la-resolver.lo -MD -MP -MF src/.deps/libstrophe_la-resolver.Tpo -c -o src/libstrophe_la-resolver.lo `test -f 'src/resolver.c' || echo './'`src/resolver.c
libtool: compile:  x86_64-w64-mingw32-gcc -DPACKAGE_NAME=\"libstrophe\" -DPACKAGE_TARNAME=\"libstrophe\" -DPACKAGE_VERSION=\"0.14.0\" "-DPACKAGE_STRING=\"libstrophe 0.14.0\"" -DPACKAGE_BUGREPORT=\"jack@metajack.im\" -DPACKAGE_URL=\"\" -DPACKAGE=\"libstrophe\" -DVERSION=\"0.14.0\" -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DSTDC_HEADERS=1 -DLT_OBJDIR=\".libs/\" -DHAVE_SNPRINTF=1 -DHAVE_VSNPRINTF=1 -DHAVE_VA_COPY=1 -DDONT_USE_GETRANDOM=1 -DHAVE_CARES=1 -DHAVE_ZLIB=1 -DLIBXMPP_VERSION_MAJOR=0 -DLIBXMPP_VERSION_MINOR=14 -I. -I. -Wall -Wextra -g -O2 -MT src/libstrophe_la-resolver.lo -MD -MP -MF src/.deps/libstrophe_la-resolver.Tpo -c src/resolver.c -o src/libstrophe_la-resolver.o
src/resolver.c:24:10: fatal error: ares.h: No such file or directory
   24 | #include <ares.h>
      |          ^~~~~~~~
compilation terminated.
make: *** [Makefile:1775: src/libstrophe_la-resolver.lo] Error 1
```

that one right?
https://github.com/c-ares/c-ares/blob/main/include/ares.h

--- a/src/conn.c
+++ b/src/conn.c
@@ -21,7 +21,7 @@
  */
 
 #include <errno.h>
-#ifndef _MSC_VER
+#if !defined(_MSC_VER) && !defined(_WIN32)
 #include <netinet/in.h>
 #endif
 #include <stdarg.h>

